autogen/main/outputs.tf.tmpl

```diff
@@ -146,6 +146,11 @@ output "service_account" {
   value       = local.service_account
 }
 
+output "subnetwork" {
+  description = "The subnetwork hosting the cluster"
```

Comment:
@NissesSenap could you clarify it will be null value if fw option is not set

Reply:
I assume that if the `data.google_compute_subnetwork.gke_subnetwork` isn't set due to https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/d252579613ca44e11ad28a59f930cdb42dea2c29/networks.tf#L22 not being set the output value will be null.

But I'm a bit unsure how terraform works here. I don't think we have a simple if statement that we can set to template the output away.
Can you do a ` count   = var.add_cluster_firewall_rules ? 1 : 0` on an output?
It seems a bit strange since it's the interface we provide to the end user.